### ISSUE TITLE
Gatsby Plugin indexing

### ISSUE THREAD

**Issue by RobinCsl** [jamesdanylik/gatsby-source-steam#1]
Hello 👋 

As part of https://github.com/gatsbyjs/gatsby/issues/14013, I would like to ask you if you could publish a patch version of your source plugin to NPM such that the keywords appear on NPM as `gatsby` and `gatsby-plugin` as documented in [Gatsby docs](https://www.gatsbyjs.org/contributing/submit-to-plugin-library/). 

At the moment, `gatsby-plugin` appears under the plural form `gatsby-plugins`. I believe you have published a version of the package that differs from the version published here on GitHub, as the `package.json` file under `node_modules/gatsby-source-steam` looks a little bit different:

```
{
 "author": {
    "name": "James Danylik",
    "email": "james.danylik@gmail.com",
    "url": "https://james.danylik.com"
  },
  "bugs": {
    "url": "https://github.com/jamesdanylik/gatsby-source-steam/issues"
  },
  "bundleDependencies": false,
  "dependencies": {
    "node-fetch": "^2.1.2",
    "query-string": "^6.1.0"
  },
  "deprecated": false,
  "description": "A GatsbyJS source plugin for building websites using Valve's Steam",
  "homepage": "https://github.com/jamesdanylik/gatsby-source-steam#readme",
  "keywords": [
    "gatsby",
    "gatsby-plugins",
    "gatsby-source-plugins",
    "steam",
    "games"
  ],
  "license": "MIT",
  "main": "index.js",
  "name": "gatsby-source-steam",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/jamesdanylik/gatsby-source-steam.git"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "version": "0.1.0"
}
```

If you need any assistance with that, please let me know how I can help.

Thank you very much!

**Comment by jamesdanylik** [comment]
Thanks for reaching out and bringing that to my attention!

You are correct; it looks like npm and the repository had fallen out of sync.  I believe I've corrected the issue in https://github.com/jamesdanylik/gatsby-source-steam/commit/654393dce115d5fab851821df8b7aaf0b561cb1f, which is published to npm as [v2.1](https://www.npmjs.com/package/gatsby-source-steam/v/0.2.1).  If you can verify this solves the problem on your end, I'll go ahead and close out this issue -- hope it wasn't too much trouble!

**Comment by RobinCsl** [comment]
Wow, thanks for being super quick and efficient! Problem solved 👍 

> hope it wasn't too much trouble

It wasn't at all. Thanks for being part of the Gatsby community 🙂 

One last thing: I don't know if that's relevant, but you removed 
```
    "steam",
    "games"
```
from the keywords list that was previously available. That means these keywords don't appear on NPM anymore.

Thanks again!
